server-side/server.py

```diff
@@ -0,0 +1,31 @@
+"""This module responds to the REST invocations from client side app."""
+
+from flask import Flask, jsonify
+from server_handler import ServerHandle
+
+app = Flask(__name__)
+
+server_prop = ServerHandle()
+
+
+@app.route('/', strict_slashes=False)
+def root():
+    """Root method which returns with Server hostname.
```

Comment:
Please add what server is for this current project, a little more description

Reply:
```suggestion
    """Root method which returns with Server hostname.
```

This is done, added a little more info to the DocString.